packages/grid/x-grid/src/tests/rows.XGrid.test.tsx

```diff
@@ -10,7 +10,9 @@ import {
   GridRowData,
   useGridApiRef,
   XGrid,
+  XGridProps,
 } from '@material-ui/x-grid';
+import { useData } from '../../../../storybook/src/hooks/useData';
```

Comment:
Makes moving and reading code easier:

```suggestion
import { useData } from 'packages/storybook/src/hooks/useData';
```